### PR TITLE
Update openstack overview

### LINKED ISSUE #3281: beta updates: updated Openstack page (build)
Copy doc: https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update the content on the page. Created three new contextual footers.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Check the page matches the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#heading=h.wl0iwha88b72)

## Issue / Card
Fixes #3281

### CODE REVIEW

File: templates/openstack/shared/_openstack_clients.html [hunk 1]
```diff
@@ -8,44 +8,32 @@ <h2 class="p-muted-heading u-align--center">A selection of our OpenStack clients
...
         <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}4ef05fb7-bestbuy.jpg" alt="Best Buy" />
       </li>
       <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}192f9755-partners-logo-nec-tagline.png" alt="NEC" />
-      </li>
-      <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}377e12ed-logo-ntt-communications.png" alt="NTT" />
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}b3315d88-walmart.svg" alt="Walmart" />
       </li>
       <li class="p-inline-images__item">
```
Comment (steverydz) [inline templates/openstack/shared/_openstack_clients.html]: NTT logo referenced in the copy doc is missing

File: templates/openstack/index.html [hunk 2]
```diff
@@ -202,12 +208,48 @@ <h2>We build you a production OpenStack in 3 weeks</h2>
...
+        <li class="p-list__item is-ticked">Canonical&rsquo;s reference architecture optimises utilisation</li>
+        <li class="p-list__item is-ticked">Design, architecture, support, operations and training in one package by Canonical</li>
+        <li class="p-list__item is-ticked">Dell, HP, Lenovo, Supermicro, Quanta servers</li>
+        <li class="p-list__item is-ticked">Multi-cloud AWS, Azure, Google, IBM, Oracle, OpenStack</li>
+        <li class="p-list__item is-ticked">Container friendly kernel makes Ubuntu #1 for kubernetes, docker and linux system containers</li>
+      </ul>
+      <p><a href="/openstack/features" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Openstack', 'eventLabel' : 'Features of Canonical OpenStack on Ubuntu', 'eventValue' : undefined });">Features of Canonical OpenStack on Ubuntu</a></p>
+    </div>
+    <div class="col-4">
+      {% include "openstack/shared/_openstack-pie-chart.html" %}
```
Comment (steverydz) [inline templates/openstack/index.html]: This graphic is different in the copy doc

**Comment by anthonydillon** [comment]
@steverydz thanks, I have added NTT and confirmed the chart is correct so have updated the image in the copy doc.

**Comment by anasereijo** [comment]
Hey @anthonydillon 

looking good but just some tiny things:

screen shot 2018-06-20 at 10 52 22

screen shot 2018-06-20 at 11 04 30

screen shot 2018-06-20 at 11 15 11

screen shot 2018-06-20 at 11 27 27

**Comment by frankiegranato** [comment]
@anthonydillon let us know when you've made Ana's updates. I'm going to wait to review this page until they are done.

**Comment by anthonydillon** [comment]
@anasereijo @frankiegranato addressed the comments above.

**Comment by anasereijo** [comment]
@anthonydillon looking good, thanks!

as discussed can you try to add the following svg to the "Real architectural flexibility" section? 

https://api.jujucharms.com/charmstore/v5/bundle/openstack-base-55/diagram.svg

thanks

**Comment by anthonydillon** [comment]
@anasereijo Added the openstack image

**Comment by anasereijo** [comment]
thanks @an
:

**Comment by pmahnke** [comment]
* in the prices for the 3 cards... the first one should be 75k, not 80
* in the button s/Learn about Canonical OpenStack Features/Learn about Canonical OpenStack features/

@anthonydillon do this and the UX will be +1

**Comment by frankiegranato** [comment]
- [ ] Use

 price-card style for the 3 cards at the top of the page.

**Comment by anasereijo** [comment]
@anthonydillon 

- new graph saved https://assets.ubuntu.com/v1/86914194-halfcostgraph.svg

- more amends:

screencapture-ubuntu-pr-3380-run-demo-haus-openstack-2018-06-22-10_20_30

**Comment by anthonydillon** [comment]
@anasereijo @pmahnke updated the page with the changes and made the new card style an include which can be found on:
- openstack
- openstack/consulting
- support/plans-and-pricing

**Comment by anasereijo** [comment]
@anthonydillon the pricing cards are looking nice, thanks! 
FYI @steverydz if you need any price cards just use this ones

**Pull request merged**